flink-staging/flink-hbase/pom.xml

```diff
@@ -112,6 +112,12 @@ under the License.
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
```

Comment:
I do not have a HBase setup here. 
Could you try to exclude all dependencies of hbase-server and add them until it works? I hope the TableInputFormat and TableOutputFormat have not too many external dependencies.